range.go

```diff
@@ -111,8 +111,8 @@ func (r *RangeOp) Fields(fieldsToRead []string) *RangeOp {
 	return r
 }
 
-// Limit sets the number of rows returned per call. If not set, a default
-// value would be applied
+// Limit sets the number of rows returned per call. A limit must be provided
+// otherwise an error will be returned.
```

Comment:
How about set a high value here, maxint for example, the server will narrow it down to 200 anyway. A value here just to prevent the error.